cmd/vclusterctl/cmd/connect.go

```diff
@@ -77,7 +78,7 @@ func NewConnectCmd(globalFlags *flags.GlobalFlags) *cobra.Command {
 	}
 
 	cobraCmd := &cobra.Command{
-		Use:   "connect [flags] vcluster_name",
+		Use:   "connect [flags] VCLUSTER_NAME",
```

Comment:
When switching to the `VClusterNameOnlyValidator` can we also change this to the `VClusterNameOnlyUseLine` variable? (Applies for below as well)

Reply:
I had just kept it as-is for better readability sake. But yes, your suggestion will make it totally in-sync usage wise with the Loft CLI.